### PR TITLE
Various cosmetic fixes to ITM_V10 format

### PR DESCRIPTION
**Pull request by critto-bg**
1) kit usability mask is split into four enums, one for each byte
2) enums for item types and weapon proficiencies are added
3) renamed unknown bit5 for the flags field at 0x18
4) "usability mask" is renamed
5) mask values for the flags field of extended header are expanded with EE-only bits

### CODE REVIEW

File: infinity/formats/itm_v1.py [hunk 1]
```diff
@@ -485,7 +628,7 @@ class ITM_V10_Format (Format):
             { 'key': 'opcode_number',
               'type': 'WORD',
               'off': 0x0000,
-              'enum': 'effects',
+              'enum': 'effects', # FIXME: this ids file does not exist
               'label': 'Opcode number'},
```
Comment (lynxlynxlynx) [inline infinity/formats/itm_v1.py:632]: these are provided by gemrb.

**Comment by lynxlynxlynx** [comment]
argh, you promised. :( Having to enumerate the commit message is a sure sign it shouldn't all be in a single commit. Other than that it looks ok.

**Comment by critto-bg** [comment]
Sorry about that. It felt appropriate to commit them all as one, the changes are miniature and mostly cosmetic. I'd accept it like that if any of my devs did it at work. Splitting individual lines into separate branches seemed excessive.

Do you want me to split them up?

File: infinity/formats/itm_v1.py (hunk 1 above)
Comment (critto-bg) [inline infinity/formats/itm_v1.py:632]: I didn't think of that. How can it be used outside of gemrb environment? I'd still add enums for game types other than gemrb.

**Comment by lynxlynxlynx** [comment]
it's a small project, so it will be fine. Little chance of cursing committers years later.

File: infinity/formats/itm_v1.py (hunk 1 above)
Comment (lynxlynxlynx) [inline infinity/formats/itm_v1.py:632]: they just need to be found, but I doubt iesh has any extra search paths one could set. The ones for ees will still need to be created, but the bg2 one can be used as a start. GemRB also has its own overrides, but that can be ignored here.

Comment (critto-bg) [inline infinity/formats/itm_v1.py:632]: Do you think that adding logic to the class' constructor that initialises the enum with an appropriate set of values depending on game type would be an acceptable solution?

Comment (lynxlynxlynx) [inline infinity/formats/itm_v1.py:632]: I thought iesh handles ids files automatically. These list are way too long to hardcode.

Comment (critto-bg) [inline infinity/formats/itm_v1.py:632]: It does. What I meant is to ship prepared files as external resource with iesh and load them up if the ids is not available in the game data (i.e., it's not gemrb).

Comment (lynxlynxlynx) [inline infinity/formats/itm_v1.py:632]: I mean the same, as the gemrb files are never in the game data folders (unless someone copies them there).

**Pull request merged**